frontend/src/components/Layout.tsx

```diff
@@ -19,6 +20,7 @@ const Layout = ({ children }: Props) => {
         {t("skip_to_main")}
       </a>
       <Header />
+      <Hero />
```

Comment:
Might be nice if the content was passed into the component here instead of hardcoded into the Hero component. That way we could reuse it on other pages if we need to in the future. 

If we did that, I think the `.text-white` class could be applied to the same `div` that sets the background color. Then all content passed in would have white text. Would still need to override that yellow "beta" tho. 

One gotcha in the future might be links. We can get around that by also adding the `.usa-dark-background` class (see [links docs](https://designsystem.digital.gov/components/link/)).

Reply:
Ooh! the `.usa-dark-background` also makes the text white.

Reply:
> Might be nice if the content was passed into the component here instead of hardcoded into the Hero component. That way we could reuse it on other pages if we need to in the future.

I generally don't optimize for future use cases like that at the time of making things, but I am happy to do it.

Reply:
> Ooh! the `.usa-dark-background` also makes the text white.

I'm not finding this to be the case, it just makes the background a dark grey and doesn't change the text color at all

Reply:
> Might be nice if the content was passed into the component here instead of hardcoded into the Hero component. That way we could reuse it on other pages if we need to in the future.

I generally don't optimize for possible futures like that, but I can

> If we did that, I think the .text-white class could be applied to the same div that sets the background color. Then all content passed in would have white text. Would still need to override that yellow "beta" tho.

Do you want `Hero` to be an empty `GridContainer` that just renders `children`? Not going to be much going on in the `Hero` component itself then. Do you anticipate we are going to have other instantiations of a Hero with a blue background and white text and entirely different children (not `h1` and text)? I'm wondering if there's a lot of usefulness to this abstraction.

> Ooh! the .usa-dark-background also makes the text white.

This is pretty confusing. Do you think that this is over optimizing for now?

Reply:
> Might be nice if the content was passed into the component here instead of hardcoded into the Hero component. That way we could reuse it on other pages if we need to in the future.

I generally don't optimize for possible futures like that. What do you think about that refactor happening when we have another Hero? We would have an empty `GridContainer` that just renders some children and not really doing much of anything else but setting a `data-testid` and some colors. The children would need to be defined elsewhere, making a level of abstraction that I'm not sure is very useful at this point, especially since future Hero components are hypothetical and we don't know what they would look like.

> Ooh! the .usa-dark-background also makes the text white.

This is pretty confusing. These `usa` classes all seem really poorly documented, and I'd not know that's why we are adding that here as a person reading the code later. I do see it in the `Link` component you shared, but specifically not clear what it's doing. Related to above, do you think that this is over optimizing for now?

Reply:
> Might be nice if the content was passed into the component here instead of hardcoded into the Hero component. That way we could reuse it on other pages if we need to in the future.

I generally don't optimize for possible futures like that. What do you think about that refactor happening when we have another Hero? Maybe it's different enough that we won't want to reuse anything. 

We would have an empty `GridContainer` that just renders some children and not really doing much of anything else but setting a `data-testid` and some colors. The children would need to be defined elsewhere, making a level of abstraction that I'm not sure is very useful at this point, especially since future Hero components are hypothetical.

> Ooh! the .usa-dark-background also makes the text white.

This is pretty confusing. These `usa` classes all seem really poorly documented, and I'd not know that's why we are adding that here as a person reading the code later. I do see it in the `Link` component code you shared, but specifically not clear what it's doing. Related to above, do you think that this is over optimizing for now?

Reply:
That's fine. You're right; we don't have to optimize this now.

Reply:
Here's what `.usa-dark-background` spits out: 

```
.usa-dark-background {
  background-color: color("base-darker");

  p,
  span {
    color: color($theme-text-reverse-color);
  }

  a {
    color: color($theme-link-reverse-color);

    &:visited {
      color: color($theme-link-reverse-color);
    }

    &:hover {
      color: color($theme-link-reverse-hover-color);
    }
  }
}
```

Looks like it only applies the reverse color to paragraphs, spans, and links. Odd that it doesn't target more elements (tho I guess it shouldn't target all children, and avoid some like inputs). We're using a `div` here, but the `span` should make the text light. 

Let's just revisit this if we ever need to add links to the hero.

Reply:
Thanks, @andycochran. Did you just dig into their repo or the package contents to grab that css?